tests/suite/cli_misc.rs

```diff
@@ -702,6 +702,36 @@ fn toolchains_symlink() {
     });
 }
 
+// issue #3344
```

Comment:
was this deliberately a comment line rather than doc line?

Reply:
Yes, this is to be coherent with other tests above and below.